doc/amazon_sagemaker_operators_for_kubernetes.rst

```diff
@@ -295,7 +295,285 @@ Your output should look like the following:
+have \ ``AmazonSageMakerFullAccess`` permissions. Confirm this when you
+install the operator.
+
+Deploy the Operator to Your Namespace
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When deploying your operator, you can use either a YAML file or Helm
+charts. 
+
+Deploy the Operator to Your Namespace Using YAML
+''''''''''''''''''''''''''''''''''''''''''''''''
+
+There are two parts to deploying an operator within the scope of a namespace. The first is the set of CRDs that are installed at a cluster level. These resource definitions only need to be installed once per Kubernetes cluster. The second part is the operator permissions and deployment itself.
+
+If you have not already installed the CRDs into the cluster, apply the CRD installer YAML using the following command:
+
+::
+
+    kubectl apply -f https://raw.githubusercontent.com/aws/amazon-sagemaker-operator-for-k8s/master/release/rolebased/namespaced/crd.yaml
+    
+To install the operator onto the cluster:
+
+-  Download the operator installer YAML using the following command:
+
+   ::
+
+       wget https://raw.githubusercontent.com/aws/amazon-sagemaker-operator-for-k8s/master/release/rolebased/namespaced/operator.yaml
+
+- Update the installer YAML to place the resources into your specified namespace using the following command:
+
+   ::
+   
+       sed -i -e 's/PLACEHOLDER-NAMESPACE/<YOUR NAMESPACE>/g' operator.yaml
+
+-  Edit the \ ``installer.yaml`` file to
+   place resources into your \ ``eks.amazonaws.com/role-arn``. Replace the ARN here with
+   the Amazon Resource Name (ARN) for the OIDC-based role you’ve created. 
+
+-  Use the following command to deploy the cluster:  
+
+   ::
+
+       kubectl apply -f installer.yaml
+
+Deploy the Operator to Your Namespace Using Helm Charts
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+There are two parts needed to deploy an operator within the scope of a namespace. The first is the set of CRDs that are installed at a cluster level. These resource definitions only need to be installed once per Kubernetes cluster. The second part is the operator permissions and deployment itself. When using helm charts you will have to first create the namespace using kubectl.
+
+
+Clone the Helm installer directory using the following command:
+
+::
+
+    git clone https://github.com/aws/amazon-sagemaker-operator-for-k8s.git
+
+Navigate to the
+``amazon-sagemaker-operator-for-k8s/hack/charts/installer/namespaced`` folder. Edit
+the \ ``rolebased/values.yaml`` file, which includes high-level parameters for the
+Chart. Replace the role ARN here with the Amazon Resource Name (ARN) for the OIDC-based role you’ve
+created. 
+
+Install the Helm Chart using the following command:
+
+::
+
+    helm install crds crd_chart/
+
+
+Create the required namespace and install the operator using the following command:
+
+::
+
+    kubectl create namespace <namespace>
+    helm install --n <namespace> op operator_chart/
+
+
+.. warning::
```

Comment:
This is not valid for the namespaced scope deployment

Reply:
Fixed